pint/testsuite/test_issues.py

```diff
@@ -817,6 +817,12 @@ def test_issue_1185(self):
             np.array((0.04, 0.09)),
         )
 
+    @helpers.requires_numpy
+    def test_issue_1250(self):
```

Comment:
I would put this in `test_numpy.py` and call it `test_linalg_norm`, but otherwise this looks good to me.

Reply:
@keewis - thanks for the feedback. I've removed this test in favour of the implementation in `test_numpy.py` ([here](https://github.com/hgrecco/pint/blob/cc56b12ca5e07ea756a71847d30635f370415344/pint/testsuite/test_numpy.py#L1429)).